paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/common/Element.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.paymentsheet.elements.common
+
+import kotlinx.coroutines.flow.Flow
+
+interface Element {
```

Comment:
Can you make this a `sealed interface`?

Reply:
This will happen here: https://github.com/stripe/stripe-android/pull/3838#issuecomment-861049326